src/boot/efi/meson.build

```diff
@@ -218,18 +218,21 @@ if have_gnu_efi
 
         libgcc_file_name = run_command(efi_cc + ['-print-libgcc-file-name']).stdout().strip()
         systemd_boot_efi_name = 'systemd-boot@0@.efi'.format(EFI_MACHINE_TYPE_NAME)
-        stub_efi_name = 'linux@0@.efi.stub'.format(EFI_MACHINE_TYPE_NAME)
+        stub_so_name = 'linux@0@.so.stub'.format(EFI_MACHINE_TYPE_NAME)
+        stub_efi_name = 'linux@0@.elf.stub'.format(EFI_MACHINE_TYPE_NAME)
```

Comment:
this doesn't look right? rhs says "efi" and lhs says "elf".

Reply:
le sigh.